Coordination meeting 2022-07-22 (#1167)
HSF/hsf.github.io: organization/_posts/2022/2022-07-21-coordination.md

---
title: "HSF Coordination Meeting #233, 21 July 2022"
layout: plain_toc
---

Present/Contributing: Graeme Stewart, Stephan Hageboeck, Stefan Roiser, Serhan Mete, Ben Morgan, Eduardo Rodrigues, Markus Diefenthaler, Allie Hall, Paul Laycock, Kyle Knoepfel, Andi Salzburger

Apologies/Contributing: Bendikt Hegner, Marc Paterno, Michel Villanueva

## News, general matters, announcements

### Analysis Ecosystems Workshop

The draft [AE2 Outcomes document](https://docs.google.com/document/d/1FChA1vob3S_rHpV-o9fRlHIaRbOrD531TEQInB9URLI/edit?usp=sharing) was circulated on Monday, open for comments until 1 August.

### Snowmass

The [US Snowmass meeting in Seattle](https://indico.fnal.gov/event/22303) is well underway. Lots of interesting talks and material to look at if you are not there in person. ([Computation Frontier Sessions](https://indico.fnal.gov/event/22303/sessions/20648/#all).)

### Future Trends in Nuclear Physics Computing Workshop

> [Future Trends in Nuclear Physics Computing](https://indico.bnl.gov/event/15089/) will be 28-30 Setember at Stony Brook University. **Save the date!**

Reviewer: ```suggestion
[Future Trends in Nuclear Physics Computing](https://indico.bnl.gov/event/15089/) will be 28-30 September at Stony Brook University. **Save the date!**
```

"Software design and development will constitute a cornerstone for the future success of the experimental program at the EIC. Moreover, the cost of the IT related components, from software development to storage systems and to distributed complex e-Infrastructures can rise considerably if a proper understanding and planning is not taken into account from the beginning in the design of the EIC. The planning must include AI and ML techniques, in particular for the compute-detector integration at the EIC, and training in these techniques. The 'Future Trends in Nuclear Physics Computing' workshop at CFNS provides an ideal forum to discuss these topics. The workshop report will provide guidance for the larger EIC community on software & computing and connect them to existing common efforts in the NP and HEP community."

> Noted that last workshops have benfited greatly from participation from HEP colleagues.

Reviewer: ```suggestion
Noted that last workshops have benefited greatly from participation from HEP colleagues.
```

## Working Group Updates

### Data Analysis

- Second meeting on analysis training in HEP [next week on 27th](https://indico.cern.ch/event/1175097/).

> - Stephan now corresponding author of metadata paper. Hoping to benefit from open-access agreement with CERN.

Reviewer: ```suggestion
- Stephan is now the corresponding author of the metadata paper. Hoping to benefit from open-access agreement with CERN.
```

### Detector Simulation

- Will restart meetings in Autumn
- Note [Geant4 Collaboration Meeting, 26-30th September](https://indico.cern.ch/event/1156193/), including public Technical Forum on afternoon of the 27th.

### Reconstruction and Software Trigger

- Topical meeting on GNNs for Tracking:

>   - Probably missed the window for the pre-summer meeting (August attendence usually low), will arrange with speaker for a September timeslot

Reviewer: ```suggestion
  - Probably missed the window for the pre-summer meeting (August attendance usually low), will arrange with speaker for a September timeslot
```

- **ACTS workshop**, September 26-30 at CERN, registration open
  - Website: <https://indico.cern.ch/event/1184037/>
  - Preliminary program
    - 2 general ACTS developer days of contributions & discussions
    - 1 tutorial day using the Open Data Detector
    - 1 day for break-outs with experiments: ATLAS, key4hep(?), EIC

>     - 1 overlfow day for topics picked up during the week (including mini-hackathons)

Reviewer: ```suggestion
    - 1 overflow day for topics picked up during the week (including mini-hackathons)
```

### PyHEP

- Preparations of [PyHEP 2022 workshop](https://indico.cern.ch/e/PyHEP2022) progressing.
  - We received a handful of abstracts so far - thanks!
  - Registration slowly approaching 600.
- No topical meeting organised over the Summer.

### Software Tools and Packaging

- Nothing new to report. There was a [meeting 5 July](https://indico.cern.ch/event/1162006/) on "Build Optimizations in ACTS" that Valentin chaired.

### Event Generators

> - Nothing new report. Next meetings in September on possible role of AI/ML for event generators (Sept. 1) and events generators in NP (Sept. 15).

Reviewer: ```suggestion
- Nothing new to report. Next meetings in September on the possible role of AI/ML for event generators (Sept. 1) and event generators in NP (Sept. 15).
```
